dowhy/causal_estimator.py

```diff
@@ -40,10 +43,14 @@ def __init__(self, data, identified_estimand, treatment, outcome,
         :param treatment_value: Value of the treatment in the treated group, for effect estimation. If treatment is multi-variate, this can be a list.
         :param test_significance: whether to test significance
         :param evaluate_effect_strength: (Experimental) whether to evaluate the strength of effect
-        :param confidence_intervals: (Experimental) Binary flag indicating whether confidence intervals should be computed.
+        :param confidence_intervals: (Experimental) Binary flag indicating whether the confidence intervals should be computed.
```

Comment:
Here too, since it is binary flag, you may consider the variable name: "estimate_confidence_interval", in line with the flag above. Also interval, not intervals.

Reply:
Please ignore this comment.

Reply:
I believe this is coupled with the causal model call. I'll leave it alone for now.